crates/solvers/src/domain/liquidity/constant_product.rs

```diff
@@ -1,6 +1,10 @@
 //! Constant product pool.
 
-use {crate::domain::eth, std::cmp::Ordering};
+use {
+    crate::domain::{eth, liquidity},
+    ethereum_types::U256,
```

Comment:
I didn't put `U256` in `domain::eth` in the `solvers` crate. My reasoning is that it is a primitive type as much as u128 is. I could move it there if you feel strongly, but it would affect more of this crate (and would likely do it in a separate PR).